### ISSUE TITLE
[Feature Request] map <CR> new line in html tag?

### ISSUE THREAD

**Issue by bwafi** [altermo/ultimate-autopair.nvim#59]
similar with  `nvim-autopairs` html tag new line when click `<CR> `

before : 

`<div>|</div>`

after `<CR>` : 
```
<div>
  |
</div>
```

**Comment by altermo** [comment]
Will be implemented when HTML tags are implemented.
HTML tags will be implemented in version 0.7 (which I am currently working on)

**Comment by bwafi** [comment]
@altermo  waah, great job, I'm really looking forward to it

**Comment by altermo** [comment]
Just note that it might take a while until I'm done with version 0.7, as I need to do a total rewrite of the whole plugin.

**Comment by bwafi** [comment]
@altermo yeah, I will wait for it, because I like several of the features in this plugin, such as tabout (without additional plugins), fast wrap, close,

**Comment by sglkc** [comment]
I currently use this as an alternative
```
{ '>', '<', newline = true , p = 11 },
```

Is there an estimation on the new version? Thanks

**Comment by altermo** [comment]
Currently, the project is in maintenance mode, there's no development on the new version, so not soon.
